playbooks/fix-duplicate-hypervisor.yml

```diff
@@ -0,0 +1,28 @@
+---
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Fix duplicate hypervisor
```

Comment:
This is not a duplicate fix, as post upgrade you have  only services left with FQDN rather than short hostnames

Reply:
Task 'Rename hypervisor node in instances table', better?

Reply:
Task 'Rename hypervisor node in instances table', better?